README.md

```diff
@@ -1,11 +1,89 @@
-# token-website
+# [token.langx.io](https://token.langx.io)
 
-Official Website for the [token.langx.io](https://token.langx.io)
+[![Better Stack Badge](https://uptime.betterstack.com/status-badges/v1/monitor/wm9t.svg)](https://status.langx.io)
 
-## Development Status
+Welcome to the LangX Token webpage! Our app, in addition to helping you learn a new language through practice and immersion with native speakers from around the world, will give you the opportunity to earn with our unique "Learn-to-Earn" initiative. This is where you can find out everything you need to know about this initiative. 
 
-Our "Coming Soon" page has been released! The full project is still under development and the first major release has not yet been launched.
+This webpage is built with HTML, CSS and JavaScript.
 
-Please stay tuned for updates and progress.
+## Table of Contents
 
-We appreciate your patience and interest.
+- [Prerequisites](#prerequisites)
+- [Getting Started](#getting-started)
+- [Development Server](#start-development-server)
+- [Issues](#issues)
+- [Stargazers](#stargazers)
+- [Contributors](#contributors)
+- [License](#license)
+
+## Setup
+
+### Prerequisites
+
+Before you begin, please make sure you have met the following requirements.
+
+- You have installed [Node.js](https://nodejs.org/en) and [npm](https://www.npmjs.com/) (Node Package Manager).
+- You have a [GitHub](https://github.com/) account
+
+### Getting Started
+
+1. Fork the repository to your own GitHub account by clicking the "Fork" button at the top right of the repository page.
+
+
+2. Clone the repository (Open your terminal and run the following commands).
+
+```bash
+# clone the repository
+git clone https://github.com/your-username/token-website.git
+
+# Make sure you replace "your-username" with your GitHub username
+```
+
+3. Navigate to the project directory
+
+```bash
+# navigate to the projects directory
+cd token-website
+```
+
+4. Install dependencies
+
+```bash
+npm install
+```
+
+## Start Development Server
+
+Once you've created your project and installed dependencies with `npm install`, you can start a development server:
+
+```bash
+npm run dev
+
+# or start the server and open the app in a new browser tab
+
+npm run dev -- --open
+```
+
+## Issues
+
+If you encounter any issues, ensure that your [Node.js](https://nodejs.org/en) and [npm](https://www.npmjs.com/) versions are up to date.
```

Comment:
Please mention here https://github.com/langx/token-website/issues to open new issues instead of Nodejs and NPM links

Reply:
Okay!